media-video/gaupol/gaupol-1.2.ebuild

```diff
@@ -35,11 +35,23 @@ DEPEND="${RDEPEND}
 
 DOCS=( AUTHORS.md NEWS.md TODO.md README.md README.aeidon.md )
 
+src_prepare() {
+	xdg_src_prepare
+}
+
+python_test() {
+	virtx py.test
+}
+
+pkg_preinst() {
+	xdg_pkg_preinst
+	gnome2_icon_savelist
+}
+
 pkg_postinst() {
-	xdg_desktop_database_update
-	xdg_mimeinfo_database_update
+	xdg_pkg_postinst
```

Comment:
Why are you forfeiting the effort developers put in calling the correct functions by using some ugly-lazy hackaround eclass?